demonstrations/utils.py

```diff
@@ -124,10 +124,14 @@ def from_env_cls(
 
     def ready_for_safetensors(self) -> dict:
         """Prepare metadata for safetensors."""
+        env_data = asdict(self.environment_data)
+        for cam in env_data["observation_config"]["cameras"]:
+            cam["resolution"] = list(cam["resolution"])
+
         return {
             "seed": json.dumps(self.seed),
             "observation_mode": json.dumps(self.observation_mode.value),
-            "environment_data": json.dumps(asdict(self.environment_data)),
+            "environment_data": json.dumps(env_data),
```

Comment:
This is my current patch. We might need a more elegant fix, maybe from the camera config side.